jenkins/automation/kubernetes/nightly-build-and-deploy.groovy

```diff
@@ -1,3 +1,8 @@
+def getBuild() {
+    buildID = sh(script: "curl -XGET 'http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/nikhildev/job/Nightly-K8s-Build-and-Deploy-test/api/json' | jq -r '.lastSuccessfulBuild.number'", returnStdout: true).trim()
```

Comment:
@shailesh-vaidya, Should we create a manual parameter to take Jenkins job path as a input instead of mentioning it in a code?

Reply:
@nikhilpatil2995 - This is the current job (Nightly Build ) name right. Can you use the env variable for the current job instead of the full job name?

Reply:
addes